src/sdk/PronunciationAssessmentConfig.ts

```diff
@@ -115,6 +115,18 @@ export class PronunciationAssessmentConfig {
         this.privPhonemeAlphabet = phonemeAlphabet;
     }
 
+    /**
+     * Sets the boolean enableMiscue property.
+     * Added in version 1.26.0
+     * @member PronunciationAssessmentConfig.prototype.enableMiscue
+     * @function
+     * @public
+     * @param {boolean} enableMiscue - enable miscue.
```

Comment:
how important is the type mismatch here for JavaScript.

input param is {bolean} got the passed data and function called take {String}.

Reply:
I understand what you're saying. How important is parity with Carbon here, as Carbon uses that backing property.